packages/fusion-runtime/src/unifiedGraphManager.ts

```
@@ -373,35 +376,47 @@ export class UnifiedGraphManager<TContext> implements AsyncDisposable {
+    );
   }
 
   public getExecutor(): MaybePromise<Executor | undefined> {
-    return mapMaybePromise(this.ensureUnifiedGraph(), () => this.executor);
+    return handleMaybePromise(
+      () => this.ensureUnifiedGraph(),
+      () => this.executor,
+    );
   }
 
   public getContext<T extends {} = {}>(base: T = {} as T) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)